compiler/circlechef/core/src/ModelChef.cpp

```diff
@@ -164,16 +164,19 @@ OpChefRegistry &op_chef_registry(void)
 }
 
 /// @brief This will prepare a set of unique builtin codes in the mode recipe
-std::set<circle::BuiltinOperator>
-gather_builtincode_set(const ::circlechef::ModelRecipe &model_recipe)
+std::map<circle::BuiltinOperator, int32_t>
```

Comment:
I'm not sure about sorting order of `set` vs `map`. Is this really OK with the file format?

Reply:
CC @parjong